khelo/lib/ui/flow/matches/match_detail/components/final_score_view.dart

```diff
@@ -37,11 +36,17 @@ class FinalScoreView extends ConsumerWidget {
       children.add(const SizedBox(height: 8));
     }
     children.add(const SizedBox(height: 8));
+    children.add(Divider(color: context.colorScheme.outline));
     if (state.match!.match_status == MatchStatus.finish) {
-      children.add(Divider(
-        color: context.colorScheme.outline,
-      ));
       children.add(_winnerMessageText(context, state.match!));
+    } else if (state.match?.match_status == MatchStatus.running) {
+      children.add(Row(
+        children: [
+          const Spacer(),
+          MatchStatusTag(
+              status: state.match?.match_status ?? MatchStatus.running),
+        ],
+      ));
```

Comment:
Can you make this row like this

```suggestion
      children.add(Row(
        mainAxisAlignment: MainAxisAlignment.end,
        children: [
          Padding(
            padding: const EdgeInsets.only(top: 16,bottom: 24),
            child: MatchStatusTag(
                status: state.match?.match_status ?? MatchStatus.running),
          ),
        ],
      ));
```